### DISCUSSION TITLE
Published Jekyll website does not apply theme

### DISCUSSION THREAD

**Discussion by tastyminerals** [community/community#21783] [Other Feature Feedback, Questions, & Ideas]
<p>I have been following <a href="https://jekyllrb.com/tutorials/video-walkthroughs" rel="nofollow noopener">https://jekyllrb.com/tutorials/video-walkthroughs</a> tutorials to create a static website.</p>
<p>I wanted to publish “minima” themed website with just one blog post.</p>
<p>In the repo settings menu I see “Your site is ready to be published at <a href="https://tastyminerals.github.io/tasty-blog/" rel="nofollow noopener">https://tastyminerals.github.io/tasty-blog/</a>.” and I follow the link.</p>
<p>What I see is</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/7df875ba65678d743dcdb396fd16ad0914849ffb.png" data-download-href="/uploads/short-url/hYo0pv72IZSRT1MLQTkIDgW0Wnx.png?dl=1" title="Capture.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/7676160/181066866-1e2fb5bc-a85c-4112-abda-371ee8a3b0b2.png" alt="Capture.PNG" data-base62-sha1="hYo0pv72IZSRT1MLQTkIDgW0Wnx" width="346" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/7df875ba65678d743dcdb396fd16ad0914849ffb_2_346x500.png, https://aws1.discourse-cdn.com/github/optimized/1X/7df875ba65678d743dcdb396fd16ad0914849ffb_2_519x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/7df875ba65678d743dcdb396fd16ad0914849ffb_2_692x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/7df875ba65678d743dcdb396fd16ad0914849ffb_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Capture.PNG</span><span class="informations">922×1329 31.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>If I build the website locally with <code>bundle exec jekyll serve</code> everything works correctly.</p>
<p>What am I missing?</p>

**Comment by tastyminerals** [comment, marked as answer]
<p>github pages need to be set <strong>baseurl: "/project_name"</strong></p>

**Comment by Theric1994** [comment]
<p><a href="mailto:theric1994@gmail.com">theric1994@gmail.com</a></p>
